src/io/import_json.cc

```diff
@@ -26,11 +26,11 @@
 
 #include <fstream>
 #include <string>
-#include <json.hpp>
+#include "json/json.hpp"
```

Comment:
As  this is an external library, should we retain angle brackets to signal that it's not really part of the OpenSCAD codebase?

Reply:
Right now, we get it from `src/ext` so we should have it with double-quotes until we move it to, say, submodules and really consider it external based on cmake configuration.

Otherwise there is an ambiguity for the compiler where to pick the file.

If we put it in angle brackets, it should be `#include <nlohmann/json.hpp>` as that is the commonly used name on systems.